javaagent/src/test/groovy/io/opentelemetry/javaagent/classloading/ShadowPackageRenamingTest.groovy

```diff
@@ -39,12 +37,12 @@ class ShadowPackageRenamingTest extends Specification {
     Class<?> clazz =
       IntegrationTestUtils.getAgentClassLoader()
         .loadClass("io.opentelemetry.javaagent.tooling.AgentInstaller")
-    URL userGuava =
-      MapMaker.getProtectionDomain().getCodeSource().getLocation()
-    URL agentGuavaDep =
+    URL userJackson =
+      JSON.getProtectionDomain().getCodeSource().getLocation()
+    URL agentJacksonep =
       clazz
         .getClassLoader()
-        .loadClass("com.google.common.collect.MapMaker")
+        .loadClass("com.fasterxml.jackson.jr.ob.JSON")
```

Comment:
I think using `io.opentelemetry.sdk.OpenTelemetrySdk` would make this more clear (my first thought was, we have jackson in the agent? until I found that this is being pulled in via the jaeger exporter)